ament_cmake_pytest/test/test_pyunicode.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+def test_print_unicode():
```

Comment:
It took me a moment to realize where this number was coming from, I think a comment would help:

```suggestion
def test_print_unicode():
    '''
    Print a non UTF-8 byte sequence in a test to verify stderr parsing.
    This value was originally reported in: https://github.com/colcon/colcon-core/issues/468
   '''
```